cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -0,0 +1,123 @@
+from asgiref.sync import async_to_sync
+from django.core.management import BaseCommand
+from django.db import transaction
+
+from cl.lib.command_utils import logger
+from cl.lib.microservice_utils import microservice
+from cl.search.models import (
+    SOURCES,
+    Court,
+    Docket,
+    Opinion,
+    OpinionCluster,
+    RECAPDocument,
+)
+
+
+def import_opinions_from_recap(court=None, total_count=0):
+    """Import recap documents into opinion db
+
+    :param court: Court ID if any
+    :param total_count: The number of new opinions to add
+    :return: None
+    """
+    count = 0
+    if not court:
+        courts = Court.objects.filter(jurisdiction=Court.FEDERAL_DISTRICT)
+    else:
+        courts = Court.objects.filter(pk=court)
+    for court in courts:
+        logger.info(f"Importing RECAP documents for {court}")
+        cluster = (
+            OpinionCluster.objects.filter(docket__court=court)
+            .exclude(source=SOURCES.RECAP)
+            .order_by("-date_filed")
+            .first()
+        )
+        if not cluster:
+            logger.info(f"No clusters for this {court}")
+            continue
```

Comment:
It would be better to exclude these courts explicitly so we know in the future which courts weren't included. 

Beyond that, in the 4-12 entmoot notes, we also said we'd exclude dcd? https://docs.google.com/document/d/1E0w_swX44qWers_1tPxu0hI9GcQtuBjc6RHksotTzw4/edit#heading=h.aysr7ee06hkb

Reply:
but it doesnt hurt to add it